ppoprf/src/ggm.rs

```diff
@@ -5,19 +5,13 @@
 
 use std::fmt;
 
-use super::PPRF;
+use super::{PPRF, PPRFError};
 use bitvec::prelude::*;
 use ring::{
     hmac,
     rand::{SecureRandom, SystemRandom},
 };
 
-#[derive(Debug)]
-enum GGMError {
```

Comment:
It feels weird for the `ggm` module to use the upstream trait's error type outside of implementing that type. Is there too much duplication if you keep `GGMError` and just impl `From` for in in PPRFError?